tests/test_wave_05.py

```diff
@@ -80,34 +75,43 @@ def test_create_goal(client):
     }
 
 
-@pytest.mark.skip(reason="test to be completed by student")
+
 def test_update_goal(client, one_goal):
-    raise Exception("Complete test")
     # Act
-    # ---- Complete Act Here ----
+    response = client.put("/goals/1", json={
+        "title": "Updated Goal Title"
+    })
+    response_body = response.get_json()
 
     # Assert
-    # ---- Complete Assertions Here ----
-    # assertion 1 goes here
-    # assertion 2 goes here
-    # assertion 3 goes here
-    # ---- Complete Assertions Here ----
+    assert response.status_code == 200
+    assert "goal" in response_body
+    assert response_body =={
+                            "goal": {
+                                "id": 1,
+                                "title": "Updated Goal Title"
+                                }
+                            }
+    goal = Goal.query.get(1)
+    assert goal.title == "Updated Goal Title"
+
```

Comment:
Good test design here! Especially with checking the DB itself in lines 95 and 96!